tests/probabilty/test_functions.py

```diff
@@ -16,6 +17,27 @@ def test_round_values_exception():
         scores.probability.functions.round_values(xr.DataArray(), -1, 5)
 
 
+@pytest.mark.parametrize(
+    ("array", "rounding_precision", "expected"),
+    [
+        (ftd.DA_ROUND, 0, ftd.EXP_ROUND1),
+        (ftd.DA_ROUND, 0.2, ftd.EXP_ROUND2),
+        (ftd.DA_ROUND, 5, ftd.EXP_ROUND3),
+    ],
+)
+def test_round_values(array, rounding_precision, expected):
+    """Tests `round_values` with a variety of inputs."""
+    output_as = scores.probability.functions.round_values(array, rounding_precision)
+    assertions.assert_dataarray_equal(output_as, expected, decimals=7)
+
+
+def test_round_values_zero():
```

Comment:
You already test rounding values with `rounding_precision=0` in the test above. Consider deleting the test as I don't think it adds anything.

Reply:
Done.